ios/ReactNativeScreenTimeAPI/ReactNativeScreenTimeAPI.swift

```diff
@@ -12,6 +12,7 @@ import ManagedSettings
 import ScreenTime
 
 import SwiftUI
+import ImageRenderer
```

Comment:
question: is this a swift package? this needs to be added to the PR so that the proper dependencies are linked